src/pipecatcloud/cli/commands/deploy.py

```diff
@@ -333,7 +333,11 @@ async def deploy(
             table,
             *
             (
-                [] if partial_config.scaling.min_instances else [
+                [
+                    Text(
+                        f"Note: Usage costs will apply for {partial_config.scaling.min_instances} reserved instance(s). Please see: https://www.daily.co/pricing/pipecat-cloud/",
+                        style="bold red",
```

Comment:
The `bold red` reads as an error at first glance. I think the same `red` styling as the cold start warning is appropriate here. 
```suggestion
                        style="red",
```